argo_workflow_tools/dsl/dag_compiler.py

```diff
@@ -485,8 +485,8 @@ def _build_dag_template(node: DAGNode, embed_workflow_templates: bool) -> argo.T
     return dag_template
 
 
-def compile_dag(entrypoint: DAGNode, on_exit: DAGNode = None,
-                embed_workflow_templates: bool = False) -> argo.WorkflowSpec:
+def compile_dag_into_workflow(entrypoint: DAGNode, on_exit: DAGNode = None,
```

Comment:
explain to me the rationale of the name change?

Reply:
talked about this yesterday f2f, the reason for this change is that the previous name implies this function compiles a dag, but it actually complies a workflow that wraps a dag